index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <title>Natalia's Portfolio</title>
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <a href="#" class="logo-placeholder">John Doe</a>
+            <nav class="ham-menu">
+                <svg width="36" height="24" viewBox="0 0 36 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd"
+                        d="M0.333496 1.58325C0.333496 1.03097 0.781211 0.583252 1.3335 0.583252H34.6668C35.2191 0.583252 35.6668 1.03097 35.6668 1.58325C35.6668 2.13554 35.2191 2.58325 34.6668 2.58325H1.3335C0.781211 2.58325 0.333496 2.13554 0.333496 1.58325ZM0.333496 11.9999C0.333496 11.4476 0.781212 10.9999 1.3335 10.9999L34.6668 10.9999C35.2191 10.9999 35.6668 11.4476 35.6668 11.9999C35.6668 12.5522 35.2191 12.9999 34.6668 12.9999L1.3335 12.9999C0.781211 12.9999 0.333496 12.5522 0.333496 11.9999ZM1.3335 21.4166C0.781211 21.4166 0.333496 21.8643 0.333496 22.4166C0.333496 22.9689 0.781211 23.4166 1.3335 23.4166H34.6668C35.2191 23.4166 35.6668 22.9689 35.6668 22.4166C35.6668 21.8643 35.2191 21.4166 34.6668 21.4166H1.3335Z"
+                        fill="white" />
+                </svg>
+            </nav>
+        </header>
+        <section class="intro">
+            <div class="name">Laura Jong</div>
```

Comment:
- [ ] In this section, be sure to use `<h1>` tag instead of the `<div>` tag. This is to ensure that semantics are adhered to.